lib/src/model/game/game_controller.dart

```diff
@@ -103,6 +103,10 @@ class GameController extends _$GameController {
 
         _socketEventVersion = fullEvent.socketEventVersion;
 
+        if (game.status == GameStatus.started && game.steps.length == 1) {
```

Comment:
The logic here should be more complicated than that.

One needs to ensure:
- you are playing this game and not spectating it
- you have not played a moved yet (so depending on the side, 0 or 1 move was played)

Reply:
> you have not played a moved yet (so depending on the side, 0 or 1 move was played)

Not sure about that one - the web UI also seems to play the sound only if no move has been played at all yet.